engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -182,6 +188,7 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
 
   when(WorkflowExecutionInProgressState) {
     case Event(BackendJobExecutionSucceededResponse(jobKey, callOutputs), stateData) =>
+      sender() ! PoisonPill
```

Comment:
Gotcha. I can remove these. (As a note though, currently the children are not poisoning themselves [PS> Not to be taken literally] )